modules/openapi-generator/src/test/java/org/openapitools/codegen/DefaultCodegenTest.java

```diff
@@ -556,14 +556,14 @@ public void testDefaultResponseShouldBeLast() {
             .addApiResponse(
                 "default", new ApiResponse().description("Default"))
             .addApiResponse(
-                "201", new ApiResponse().description("OK"))
+                "422", new ApiResponse().description("Error"))
```

Comment:
@thiagoarrais I notice that you've updated the test to replace 2xx with 4xx status code.

Does it mean the original "201" status code test fail?

Reply:
No. 422 just seemed more realistic. 201 is a success and more likely to be equated to the default case.